multisense_ros/src/camera.cpp

```diff
@@ -1927,6 +1927,35 @@ void Camera::timerCallback()
     }
 }
 
+crl::multisense::Status Camera::verifyFpsChange(const crl::multisense::image::Config& config)
```

Comment:
@psmithcrl Is FPS the only parameter which needs to be confirmed like this? Out of curiosity what failure were you seeing by setting the FPS once?

Reply:
Found the thread on slack. Sorry